shell/client/shell.html

```diff
@@ -183,6 +176,28 @@
   </ul>
 </template>
 
+<template name="notificationItem">
+  <div class="notification-item">
+    <div class="notification-header">
+      {{#if isAppUpdates}}
+        <span>App Market has updated apps</span>
```

Comment:
How about just saying "App updates are available" -- specifying "App Market" could be incorrect for people using an in-house app index.